infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -240,6 +248,10 @@ type PvcCreate struct {
 type Registry struct {
 	Local  *LocalRegistryConfig  `json:"local,omitempty"`
 	Remote *RemoteRegistryConfig `json:"remote,omitempty"`
+	// LogLevel sets the logging level for the registry service
+	// Allowed values: "debug", "info", "warning", "error", "critical".
+	// +kubebuilder:validation:Enum=debug;info;warning;error;critical
+	LogLevel string `json:"logLevel,omitempty"`
```

Comment:
this should be moved within the `Local` object.

Reply:
sure, will do just for my understanding what if user decides to use RemoteRegistryConfig? how log will be set.

Reply:
change done.

Thanks for pointing me in the right direction. I had missed the concept of RemoteRegistry.